charts/cortx/templates/_cluster.yaml.tpl

```diff
@@ -3,7 +3,7 @@
 - name: {{ .name }}
   {{- if eq .type "server_node" }}
   id: {{ required "A valid id is required for server nodes" .id | quote }}
-  {{- else if eq .type "data_node" }}
+  {{- else if (or (eq .type "data_node") (eq .supertype "data_node")) }}
```

Comment:
```suggestion
  {{- else if hasPrefix "data_node" .type }}
```
**suggestion:** What if we just use a consistent prefix name for all data nodes, instead of requiring an additional parameter? You could name them `data_node/1` ... `data_node/106`, which would match the provisioner sample.

As of now, the name is based on the release name, which doesn't seem entirely necessary?

Or, perhaps I am missing some significance to the naming?